src/exts/tvdb_info/main.py

```diff
@@ -87,24 +89,40 @@ def __init__(self, bot: Bot) -> None:
         choices=["movie", "series"],
         required=False,
     )
+    @option("by_id", input_type=bool, description="Search by tvdb ID.", required=False)
     async def search(
-        self, ctx: ApplicationContext, query: str, entity_type: Literal["movie", "series"] | None = None
+        self,
+        ctx: ApplicationContext,
+        query: str,
+        entity_type: Literal["movie", "series"] | None = None,
+        by_id: bool = False,  # noqa: FBT001, FBT002
```

Comment:
Maybe theese noqa aren't optimal here...

Reply:
You can avoid them by turning `by_id` into a keyword-only argument